paimon-web-ui/src/store/job/index.ts

```diff
@@ -79,10 +79,10 @@ export const useJobStore = defineStore({
       return (key) => {
         const initResultData = state.jobs[key]?.job?.resultData
         const refreshedResultData = state.jobs[key]?.jobResultData?.resultData
-        if (initResultData)
+        if (initResultData && initResultData.length > 0)
```

Comment:
I don't think this should add this condition.

Reply:
If this condition is not added, the result is not as expected, and the rows of the query result data are 0.

Reply:
@s7monk, could you explain the reason why need to add this condition?

Reply:
The type of initResultData is ResultDataItem[] | undefined. If only if (initResultData) can only judge the case where it is not undefined, this branch will be entered when initResultData is an empty array, so initResultData.length is returned, which is exactly 0. This is my understanding.

![image](https://github.com/apache/paimon-webui/assets/34889415/fabba470-11cf-4263-8907-3d6eafc44cea)